javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/JavaParserTypeSolver.java

```diff
@@ -142,25 +156,26 @@ public void setParent(TypeSolver parent) {
         String[] nameElements = name.split("\\.");
 
         for (int i = nameElements.length; i > 0; i--) {
-            String filePath = srcDir.getAbsolutePath();
+            StringBuilder filePath = new StringBuilder(srcDir.getAbsolutePath());
```

Comment:
Better...

Reply:
The yellow warnings of Intellij started to annoy me :-)